webgpu/README.md

```diff
@@ -0,0 +1,10 @@
+# WebGPU Conformance Test Suite
+
+The WebGPU CTS requires support for the WebGPU API. This requires both browser
+support and hardware support, so this API cannot run on most automated testing
+infrastructure. Tests inside this directory should always be skipped if
+appropriate GPU hardware is not available.
+
+The contents of this directory are automatically generated from TypeScript
+sources which live upstream in the [WebGPU CTS](https://github.com/gpuweb/cts).
+They are periodically built and pushed to WPT.
```

Comment:
How will you deal with changes made in the wpt repository, either directly or through synchronization from browser repositories?

Reply:
Since these files are all generated from an upstream source repo, changes would be synced from there into WPT directly, probably via an automated process each time a PR is merged upstream. Does this seem like a reasonable workflow?

Reply:
(And this readme and the file headers would instruct people not to make changes directly in WPT or in the browser repo copies of WPT.)

Reply:
@Ms2ger Do you have any comments about this?